.github/workflows/update-status-chart.yml

```
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Update Status Chart
+on:
+  schedule:
+    - cron: "0 7 * * *"
+  workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout gh-pages
+        uses: actions/checkout@v2
+        with:
+          ref: gh-pages
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: ">=15.12.0"
+      - name: Run gather_stats.js
+        run: |
+          export SECRET_GITHUB_PERSONAL_ACCESS_TOKEN=${{ github.token }}
+          npm ci
+          node gather_stats.js

```

Paranoia: I suggest putting the PAT only in the environment for `node`:
```suggestion
          npm ci
          SECRET_GITHUB_PERSONAL_ACCESS_TOKEN="${{ github.token }}" node gather_stats.js
```
which is still an enormous amount of exposure, but slightly less enormous.